sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample06_Transactions.md

```diff
@@ -44,6 +44,48 @@ using (var ts = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled))
 }
 ```
 
+### Transactions across entities
+In the previous version of the library, when attempting to perform transactions that spanned multiple entities, it was necessary to configure your senders so that they would send-via the other entity in the transaction. With updates to the service, the new library introduces the concept of transaction groups to handle cross entity transactions. When creating senders and receivers that should be part of a cross-entity transaction, you can specify the same `TransactionGroup` string in the options as shown below. 
+
+There are a few caveats when working with transaction groups: 
```

Comment:
To use this header-style format, you'd need either a list to follow or a series of paragraphs showing a sequence  _(e.g.  First blah blah.   Next blah blah.  Finally blah blah.)_    Since you seem to have a single, cohesive paragraph, you may want to consider just folding this in and making it the leading sentence in the paragraph itself.